Modules/_fuzz/fuzzer.cpp

```diff
@@ -0,0 +1,118 @@
+// A fuzz test for CPython.
+//
+// Unusually for CPython, this is written in C++ for the benefit of linking with
```

Comment:
I added a small note, but it's probably insufficient. Fortunately, we do link to the oss-fuzz docs if people are confused.